src/main/java/com/minecolonies/blockout/views/ScrollingView.java

```diff
@@ -4,6 +4,7 @@
 import com.minecolonies.blockout.PaneParams;
 import com.minecolonies.blockout.controls.Scrollbar;
 import org.jetbrains.annotations.NotNull;
+import org.lwjgl.input.Mouse;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.lwjgl.input.Mouse'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)